packages/admin/admin/src/table/TableDndOrder.tsx

```diff
@@ -104,27 +108,77 @@ function DndOrderRow<TRow extends IRow>(props: IDndOrderRowProps<TRow>) {
 
     return (
         <TableBodyRow ref={refRow} {...rowProps} style={{ opacity }}>
-            <TableCell ref={refDragHandle} className={classes.dragCell}>
-                <div className={classes.dragIconContainer}>{props.dragHandleIcon}</div>
-            </TableCell>
+            <DragCell ref={refDragHandle} {...slotProps?.dragCell} {...restProps}>
```

Comment:
Why are the `restProps` now being added to the cell? What do they contain?

Reply:
You're right, it's of no use there